lawlibrary/templates/peachjam/about.html

```diff
@@ -0,0 +1,41 @@
+{% extends "peachjam/layouts/main.html" %}
+{% load i18n %}
+
+{% block title %} {% trans 'About' %}{% endblock %}
+
+{% block page-content %}
+<section class="py-5">
+  <div class="container">
+    <h1 class="mb-4">{% trans 'About the platform' %}</h1>
+    <p>
+      {% blocktrans %}
+      Company copy
+      {% endblocktrans %}
+    </p>
+
+    <iframe
+        src="https://docs.google.com/forms/d/e/1FAIpQLScrIGxy1Qs-lb4FA1myX1uv1u_pCAVUJ6as_w4QbQLmCJZK7Q/viewform?embedded=true"
+        class="mb-4"
+              width="100%"
+              height="800px" frameborder="0" marginheight="0" marginwidth="0">Loading…
+    </iframe>
+
+    <h2 class="mb-4">{% trans 'Our partners' %}</h2>
```

Comment:
I think you can drop this, since all the logos are in the footer?

Reply:
done